pkg/jwx/models.go

```diff
@@ -11,7 +11,7 @@ type DecodedAccessTokenHeader struct {
 
 // Claims served by keycloak inside the accessToken
 type Claims struct {
```

Comment:
StandardClaims has been deprecated for a while, and was finally removed in v5. RegisteredClaims is the new equivalent.